exercises/mongo-shell/basic-mongo-queries.md

```diff
@@ -1,7 +1,157 @@
-## READ Operations
+# CRUD Operations
+
+This document deals with basic create, read, update and delete operations using the mongo shell. 
+
+[Read more](https://docs.mongodb.com/manual/crud/) :book: about the CRUD operations here. 
+
+## Create Operations
+
+A MongoDB database is made up of 1 or more collections which are in turn made up of 1 or more documents. A document has a `key: value` structure, similar to JSON objects. In the following exercises, we'll learn how to insert documents using the following operations:
+* `insertOne()`
+* `insertMany()` 
+
+**Exercise 1** :computer: 
+
+* Verify what database you are currently using (should be `test`, this is the default.)
+
+    ```javascript
+    db
+    ```
+
+* Let's create a new database called `employee`. 
+    ```javascript
+    use employee_db
+    ```
+
+:arrow_right: Note that `use` creates a new database, if one does not already exist. 
+
+* Next, let's create a collection called `employee_info`.
```

Comment:
How do we do so?  How is this similar/different from SQL which they have been writing the entire time?

Reply:
look at the rest of the suggestions and let me know what you think

Reply:
🗑

Reply:
:fu: